android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'co.edgesecure.app'
+include ':react-native-in-app-review'
+project(':react-native-in-app-review').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-in-app-review/android')
```

Comment:
This should not be needed thanks to autolinking. Can you delete it and check that the feature still works?